tests/test_spend.py

```diff
@@ -386,8 +386,8 @@ def test_spends_conflicting(revault_network, bitcoind):
 @pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
 def test_spend_threshold(revault_network, bitcoind, executor):
     CSV = 20
-    managers_threshold = 3
-    revault_network.deploy(17, 8, csv=CSV, managers_threshold=managers_threshold)
+    managers_threshold = 2
```

Comment:
*Comment about the commit message, not this line.*

> I'm not really sure why we would test with a deployment that
big (I'm fairly convinced it was just bad copypasta from another
test). This should reduce the amount of timeouts in CI.

My rationale was testing larger setups. It should be an explicitly dedicated test, instead so ACK for decreasing it.